why_postgres.mdx

```diff
@@ -0,0 +1,124 @@
+
+The Proxy allows us to "virtualize" access not only to the branch, but also to the Postgres catalog of tables, functions, sequences, and so on. ORMs like Prisma and Drizzle do fairly complex introspection to see the tables and columns that are currently defined, and we wanted to make sure they work without any modifications.
+
+You might be wondering why we chose to represent database branches as Postgres schemas rather than Postgres database. The reason has to do with logical replication, which is a key component of the data platform, because reliable change-data-capture (CDC) is such an important feature keeping multiple data stores in sync. Individual databases would have forced us to do a replication slot for each.
+
+#### Logical replication events
+
+Speaking of logical replication, another challenge that we had to solve was getting not only data events but also schema events from Postgres. A well known limitation of the Postgres logical replication system is that it only includes data changes, no DDL changes. Yet, having schema changes in the same stream of events simplifies significantly the downstream consumers for the logical replication events. The way we solve this is by installing [DDL event triggers](https://www.postgresql.org/docs/current/event-triggers.html) that are writing the schema changes to an "out-box" table, which gets replicated via the same replication slot.
+
+We think this tech will be useful for many Postgres applications, so we are planning to open source it soon as its own project.
+
+#### Integrated pgroll
+
+Finally, we have integrated [pgroll]() in our platform. pgroll is our open-source project for zero-downtime, reversable schema migrations. While `pgroll` is CLI tool, in order to integrated it seamlesly in the Xata platform, we have developed a server API for it, which integrates with our control plane. For now, this allows us to keep track of all DDL statements that are being executed and opens the door to a much better schema-changes workflow. Look out for a detailed blog post this week about pgroll and how we integrated it into Xata.
+
+## Get started
+
+The Xata Postgres service is currently in public Beta. You can opt-in to the Beta by flipping a flag in your [workspace's settings](). Once you have enabled the flag, when creating a new database, you can select whether you want to have Postgres direct access or not. Because the underlying changes are significant, at the moment, it's not possible to migrate existing Xata database to the Postgres kind. However, we are planning to provide a migration path during the Beta period.
```

Comment:
```suggestion

## Get started

The Xata Postgres service is currently in public Beta. You can opt-in to the Beta by flipping a flag in your [workspace's settings](). Once you have enabled the flag, when creating a new database, you can select whether you want to have Postgres direct access or not. Because the underlying changes are significant, at the moment, it's not possible to migrate existing Xata database to the Postgres kind. However, we are planning to provide a migration path during the Beta period.

## Learn more
Want to learn more about the development of this service, the folks that built it and watch a quick demo of it in action? Check out our latest meet the makers session here.

<ArticleVideo platform="youtube" src="https://www.youtube.com/embed/d-u2Naheqc4?si=dx9ZfGYTzD95-h26" />

Pop into [Discord](https://xata.io/discord) and say hi if you'd like to dig in further 👋 
```

@CezzaineZ I'm sure there's room for improvements, but what do you think about this in general? I thought it might make sense to have at the bottom. Open to any and all suggestions, but wanted to drop in so we don't forget. The YouTube videos are currently unlisted.